cblox/src/core/tsdf_esdf_submap.cpp

```diff
@@ -9,7 +10,41 @@ void TsdfEsdfSubmap::generateEsdf() {
                                           esdf_map_->getEsdfLayerPtr());
   // Generate the ESDF
   LOG(INFO) << "Generating ESDF from TSDF for submap with ID: " << submap_id_;
+  esdf_integrator.setFullEuclidean(false);
   esdf_integrator.updateFromTsdfLayerBatch();
 }
 
+void TsdfEsdfSubmap::setTsdfMap(const voxblox::Layer<TsdfVoxel>& tsdf_layer) {
+  tsdf_map_.reset(new voxblox::TsdfMap(tsdf_layer));
+}
```

Comment:
Where do we use this function? Are there times where we actually switch a submap's layer after it's creation.

The issue with this is that if we do need this, we open the door to inconsistencies in the class. For example the esdf and the tsdf being mismatched in some way.

Reply:
Yeah I agree, this seems a bit dangerous, I guess I will just remove it for now and in case there is a use-case for it we can bring it back.

Reply:
Cool!